packages/components/src/StatusLabel/StatusLabel.css

```diff
@@ -1,20 +1,58 @@
-:root {
-  --statusLabel-icon-diameter: 10px;
-}
-
 .statusLabelRow {
   display: flex;
+  width: fit-content;
+  padding: 6px 10px 6px 8px;
+  border-radius: 12px;
   flex-direction: row;
   flex-wrap: nowrap;
-  gap: var(--space-small);
-  width: fit-content;
+  gap: 6px;
+}
+
+/* Reset the <Text> line height so we can reliably get 24px tall */
+.statusLabelRow p {
```

Comment:
Is there a better way to do this using Typography rather than Text?

Reply:
I found setting the height of `.statusLabelRow` to `12px` (24px - paddingTop - paddingBottom) and removing the lineHeight got the height of the status label to reliably get to 24px as well. Up to you on that though (I didn't do extensive testing)

Reply:
@MichaelParadis yeah challenge there will be in some cases the labels will wrap onto two lines which will throw the height off.

Reply:
That makes sense. In that case I think the lineHeight stuff makes the most sense